src/test/test_deployer.py

```diff
@@ -121,13 +122,25 @@ def test_deployer_config(self, app, deployer):
         assert deployer.template_path == Path(
             'cloudlabs/deployer/terraform').absolute()
 
+# @pytest.mark.parametrize('sku', ['14.04.5-LTS',
+#                                  '16.04-LTS',
+#                                  '17.10']):
+# @pytest.mark.parametrize('vm_type', ['Standard_A6', ''])
+# def test_deploy_ubuntu_vm(self, sku):
+
+
     # TODO: Choose a suitable list of machines and OS.
     @pytest.mark.parametrize("vm_type",
-                             ["Standard_A" + str(i) for i in range(5, 8)])
+                             ["Standard_A7"])
```

Comment:
I'm guessing this is just to limit the number of tests?
By the way, [parameterising the fixture function]( https://docs.pytest.org/en/latest/fixture.html#fixture-parametrize) (rather than the test) should also be doable, if I understand right?

Reply:
You are right. Thanks! I just did no research at all on this 0:) For now I'm just trying to make the tests pass in Travis without them taking ages (that's why I'm just trying with one `vm_type` as well. This is still very much work in progress.

Also I'm trying to find out which are the possible `vm_size` values I can give to Terraform, the documentation is not clear about this. The promo machines might be better than this one, or at the very least they are cheaper!

Reply:
List returned by a TF error: https://github.com/UCL-CloudLabs/user-portal/wiki/List-of-VM-types

Reply:
I remember now the problem about parametrising fixtures. It is OK as long as you just one to parametrise one. If you need to combine several like I was trying (vm size and OS), it became overcomplicated. I think it needs helper functions to generate the tests. So for now it's worked out because I broke down the things I need to parametrise in different fixtures that only need on param. I didn't do it to work around it, but because it made sense. So done!